src/types/Person.ts

```diff
@@ -0,0 +1,14 @@
+export interface PersonFromServer {
+  'name': string,
+  'sex': string,
+  'born': number,
+  'died': number,
+  'fatherName': string,
+  'motherName': string,
+  'slug': string,
```

Comment:
redundant `string` literals `''`, you can remove them

Reply:
Oops...just copied this from API and hadn't changed😅